### ISSUE TITLE
Cannot find module - Nextjs

### ISSUE THREAD

**Issue by matheusdamiao** [baptisteArno/typebot.io#1738]
**Describe the bug**

I'm running a Nextjs project locally.
Whenever I start the development server, it first throws the following error message, but when I refresh Typebot is loaded and is working perfectly. 

I'm using :
```
"@typebot.io/js": "^0.3.11",
"@typebot.io/nextjs": "^0.3.11",
```

![image](https://github.com/user-attachments/assets/156ff6dd-e1f3-4264-b4bb-6241e3796742)

and on my Console, it says:

![image](https://github.com/user-attachments/assets/90fb7cd7-8ed6-4fb3-986f-00c48c40c1d1)

I searched on the issues before submiting this one, but didn't find any similar error.

Any idea on how I can fix it?

Thanks in advance for any help and big thanks for this amazing project.

**Comment by baptisteArno** [comment]
So you have this error when running your dev command but then you refresh the page and it all works?

**Comment by matheusdamiao** [comment]
exactly! First loading in dev server it crashes throwing the error above and if I refresh, everything works perfectly.

I also tried to build the project and it throws an error as well.

**Comment by matheusdamiao** [comment]
I'm gonna close this issue as I've tried to reproduce it in a brand new nextjs project and the error is not taking place there, 
so the problem seems to be related to my own project and not to Typebot.
